src/test/java/com/yahoo/bullet/querying/QuerierTest.java

```diff
@@ -490,6 +491,47 @@ public void testBasicWindowMaximumEmittedWithNonMatchingRecords() {
         Assert.assertNull(querier.getData());
     }
 
+    @Test
+    public void testLogicFilterNot() {
```

Comment:
Let's add a test for what happens when you mix and match ObjectFilterClause and StringFilterClause (both cases when first is String or Object). We parse by only looking at the first value. Are the other filters ignored? Are there RunTimeExceptions?

Reply:
They are RunTimeExceptions. Added 2 tests.